templates/files/conf/k8s-addons

```diff
@@ -128,9 +128,17 @@ $KUBECTL -n kube-system delete deploy calico-kube-controllers --ignore-not-found
 # delete legacy nginx daemonset
 $KUBECTL -n kube-system delete daemonset nginx-ingress-controller --ignore-not-found=true
 
+# delete coredns manifests
+$KUBECTL -n kube-system delete svc coredns --ignore-not-found=true
+$KUBECTL -n kube-system delete serviceaccount coredns --ignore-not-found=true
+$KUBECTL -n kube-system delete cm coredns --ignore-not-found=true
+$KUBECTL -n kube-system delete deploy coredns --ignore-not-found=true
+$KUBECTL delete clusterrolebinding system:coredns --ignore-not-found=true
+$KUBECTL delete clusterrole system:coredns --ignore-not-found=true
+
```

Comment:
what happens if this is rerun after the app is installed?

Reply:
maybe some if that will check if the app exists and delete only if it does not exists yet?

Reply:
very good point. I thought of this while doing nginx ingress controller (PR coming), didn't check for coredns.
Checking

Reply:
how about this?

Reply:
looks good if it works as expected :p